cmd/common/config.go

```diff
@@ -217,28 +245,16 @@ func BuildConfigKeyWithUser(keyPartial, userID string) string {
 }
 
 func isTokenExpired(bearerToken string) bool {
-	token, err := jwt.Parse(bearerToken, func(_jwtToken *jwt.Token) (interface{}, error) {
-		// uncomment when enabling local verification
-		// var kid *string
-		// if kidhdr, ok := _jwtToken.Header["kid"].(string); ok {
-		// 	kid = &kidhdr
-		// }
-
-		// publicKey, _, _, _ := util.ResolveJWTKeypair(kid)
-		// if publicKey == nil {
-		// 	msg := "failed to resolve a valid JWT verification key"
-		// 	if kid != nil {
-		// 		msg = fmt.Sprintf("%s; invalid kid specified in header: %s", msg, *kid)
-		// 	} else {
-		// 		msg = fmt.Sprintf("%s; no default verification key configured", msg)
-		// 	}
-		// 	return nil, fmt.Errorf(msg)
-		// }
-
-		return nil, nil
-	})
+
+	var jwtParser jwt.Parser
+	token, _, err := jwtParser.ParseUnverified(bearerToken, jwt.MapClaims{})
+	if err != nil {
+		log.Printf("failed to parse JWT token on behalf of authorized user; %s", err.Error())
+		os.Exit(1)
+	}
 
 	if err != nil {
+		log.Printf("isTokenExpired err: %s", err)
```

Comment:
Not sure if this is quite "correct" as it still returns that the token hasn't expired if there is an error.

Reply:
Can we get a fix for this?